modules/Lending.py

```diff
@@ -52,8 +52,10 @@ def init(cfg, api1, log1, data, maxtolend, dry_run1, analysis, notify_conf1):
     notify_conf = notify_conf1
 
     global sleep_time, sleep_time_active, sleep_time_inactive, min_daily_rate, max_daily_rate, spread_lend, \
-        gap_bottom_default, gap_top_default, xday_threshold, xdays, min_loan_size, end_date, coin_cfg, min_loan_sizes, dry_run, \
-        transferable_currencies, keep_stuck_orders, hide_coins, scheduler, gap_mode_default
+        gap_bottom_default, gap_top_default, xday_threshold, xdays, min_loan_size, end_date, coin_cfg, min_loan_sizes, \
+        dry_run, transferable_currencies, keep_stuck_orders, hide_coins, scheduler, gap_mode_default, exchange
+
+    exchange = api.__class__.__name__.upper()
```

Comment:
What's wrong with getting it from config? You already have it when you initialise from lendingbot:
```
47 exchange = Config.get_exchange()
```
 and we shouldn't use two different methods to work out what exchange we're using.

It should really be part of the init (again, multiple exchanges at once), but the class needs refactored before then.